docs/references/phasing.tex

```diff
@@ -1,111 +1,179 @@
+    \label{fig:simple_interferometer}
+\end{figure}
 
-However, there is a additional relativistic effect that depends on the velocity of the antennas at the time of observation. An observer at the barycenter of the solar system will see the distance between the antennas relativistically shorten and rotate (Terrell rotation) due to the velocity of the Earth around the solar system (and to a lesser degree the movement of the antennas due to Earth rotation). This effect is known as `aberration' and is a 20" scale effect over the course of a year. From the radio perspective, if we define the location of distant AGN relative to the solar system barycenter (so they don't move with time during a year), then the baseline is shortened and rotated in this barycentric frame relative to the locations on the ground due to the special relativistic transform between the frames.\footnote{The barycenter is actually not an inertial frame due to motion of the solar system around the galaxy, and this is not a small effect (e.g. 200"). However, this is hard to measure and changes slowly, so the barycenter is assumed to be inertial. But travelers over StarTrek distances should beware.} 
+\begin{figure}[!t]
+    \centering
+    \includegraphics[width=0.45\textwidth]{ncp_icrs_app.png}
+    \ 
+    \includegraphics[width=0.45\textwidth]{app_icrs_frame_comp.png}
+    \caption{\emph{Left}: Figure showing the region around the North Celestial Pole (NCP; $\delta=90^{\deg}$) of both ICRS (red) and apparent (black) coordinates, approximately 0.5$^{\deg}$ across. Solid lines show lines of constant right ascension, and dashed lines show lines of constant declination. In this region, the direction of the NCP -- which determines the direction of North on the map -- can change dramatically when considering the ICRS versus apparent NCP. If comparing interferometric maps to results in an ICRS frame, it is then useful to rotate in the map -- either in the image domain or in the $uv$-domain, so that North is consistent. \texttt{pyuvdata} takes the latter approach of rotating the $uv$-coordinates so that the resultant image is appropriately oriented with the coordinate frame of the source -- be it ICRS, FK5, or other supported frame. \emph{Right}: Further away from the NCP, lines of constant right ascenscion and declination are closer to parallel, with the typical angle between them being a few tenths of a square degree -- a relatively small effect, which can be significant for larger maps (i.e., where the longest baselines are $>> 100\times$ in length versus the antenna diameter).}
```

Comment:
"it is then useful to rotate in the map" is awkward. Maybe: "it is useful to rotate the map"

Also, I think this goes beyond "useful". I think it is required if you want to make meaningful maps -- either to be able compare them to catalogs or other telescopes or to be able to combine data in the image space (which is what we do with w-snapshots).

Reply:
"which can be significant for larger maps (i.e., where the longest baselines are $>> 100\times$ in length versus the antenna diameter).}" This confuses me. Are you talking about large uv planes (long baselines so good angular resolution) or about large maps (large field of view, so small antennas)?

Reply:
Removed the "useful" and stated "one needs to rotate the map so that the direction of North is consistent". I removed the last comment you brought up here -- my comment here was highlighting that the impact of this is felt when your map is many resolution elements across (e.g., not just a low-res, small FoV), but this is a pretty subtle nuance and I think it's probably not a helpful aside in a figure caption.